src/convertToObject.js

```diff
@@ -6,7 +6,24 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const styleArray = sourceString.replace(/^\n+/g, '').split(';');
+  const data = {};
+
+  styleArray.forEach((elem) => {
+    const style = elem.trim();
+
+    if (!style) {
+      return;
+    }
+
+    const [key, value] = style.split(':').map((item) => item.trim());
```

Comment:
You can simplify like this
```suggestion
    const [key, value] = elem.split(':').map((item) => item.trim());
```